README.md

```diff
@@ -1,72 +1,93 @@
-  - `fn: `[`<Function>`]` | `[`<RegExp>`] to be searched
-  - `text: `[`<string>`] to be searched in
-  - `start: `[`<number>`] position to start searching from
+  - `fn`: [`<Function>`]` | `[`<RegExp>`] to be searched
+  - `text`: [`<string>`] to be searched in
+  - `start`: [`<number>`] position to start searching from
 
 *Returns:* [`<Object>`] function signature
-  - `title: `[`<string>`] short function description
-  - `description: `[`<string>`] extended function description
-  - `parameters: `[`<string[]>`][`<string>`] function parameters
-  - `comments: `[`<string[]>`][`<string>`] comments about returned value, thrown
-        errors, deprecation and usage
+  - `title`: [`<string>`] short function description
+  - `description`: [`<string>`] extended function description
+  - `parameters`: [`<Object[]>`][`<Object>`] function parameters, { name, types,
+        nonStandardTypes, comment, offset }
+  - `comments`: [`<Object[]>`][`<Object>`] comments about returned value, thrown
+        errors, deprecation and usage, { name, types, nonStandardTypes, comment
+        }
 
 
-### Generate md from interfaces inventory
+Parse function signature
 
-`generateMd(inventory)`
-  - `inventory: `[`<Map>`] hash of map of records, { method, title, parameters }
+
+#### generateMd(inventory, options)
+
+  - `inventory`: [`<Map>`] hash of map of records, { method, title, parameters }
+  - `options`: [`<Object>`]
+    - `header`: [`<string>`] text before api documentation
+    - `footer`: [`<string>`] text after api documentation
+    - `customTypes`: [`<string[]>`][`<string>`] custom types
+    - `customLinks`: [`<Array[]>`][`<Array>`] custom types links
 
 *Returns:* [`<string>`] md document
 
 
+Generate md from interfaces inventory
+
+
+## Contributors
+
+See GitHub for a full [list of contributors](https://github.com/metarhia/metadoc/graphs/contributors)
+
+## License
+
+Licesed under MIT license. Copyright (c) 2018 Metarhia contributors
+
+
 [`<Object>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
-[`<Date>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date
 [`<Function>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function
 [`<RegExp>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
-[`<DataView>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView
 [`<Map>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
-[`<WeakMap>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap
-[`<Set>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set
-[`<WeakSet>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet
-[`<Array>`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
```

Comment:
This shouldn't probably be removed since it is used on line 71 for `customLinks` argument.